app/models/pool.rb

```diff
@@ -10,7 +10,7 @@
 #
 
 class Pool < ActiveRecord::Base
-  validates :tournament_id, presence: true
+  validates :tournament_id, :name, presence: true
```

Comment:
We should also do this (and other validations) at the database level with migrations. Open an issue, please.